.github/workflows/build.yaml

```diff
@@ -39,7 +39,7 @@ jobs:
 
       - name: Build and push Docker image
         run: |
-          make ko-push IMG=${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.VERSION }},latest
+          make docker-buildx IMG=${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.VERSION }}
```

Comment:
Is there something that would work with podman too?

I guess that would bring us into buildah territory,

Reply:
It's probably fine, docker-buildx seems to be available in fedora at least

Reply:
`make docker-build` works with podman, but it only produce image for a single architecture. For releasing to quay.io we might still want to use `make docker-buildx`